CDCgov/prime-reportstream
# Convert details page to new API tools (Fix admin 401)

This PR removes the old `rest-hooks` usage from `/details` and solves the 401 caused by `ignore` being passed as the `organization` header.

Test Steps:
1. Load RS
2. Mock a receiver like `md-phd` from the Admin screen
3. Navigate to `/daily-data`. If no data is present, [this helped me populate my data](https://www.slack.com/archives/C01HFHPNJEL/p1657030728655589?thread_ts=1656355073.593839&cid=C01HFHPNJEL)
4. Click the ID of a report to get to the details page
5. Ensure no non-`2xx` calls are returning and the page loads successfully!

## Changes
<img width="357" alt="Screen Shot 2022-08-02 at 11 19 32 AM" src="https://user-images.githubusercontent.com/4022304/182411488-cf0ee9f8-3094-4bca-b9d0-a975a9863658.png">
<img width="1248" alt="Screen Shot 2022-08-02 at 11 19 16 AM" src="https://user-images.githubusercontent.com/4022304/182411554-2673ec08-e12f-4e69-9430-2f835d7919f2.png">


## Checklist

### Testing

### Process

## Linked Issues
- Fixes #6126 

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
